user/pages/03.commerce2/02.developer-guide/02.install-update/02.installation/docs.md

```diff
@@ -41,13 +23,10 @@ Requirements
 
  Tips:
 
-   The `bin` folder contains [Drupal Console] binary.
-   The `web` folder represents the document root.
-   Composer commands are always run from the site root (`mystore` in this case).
-   Downloading additional modules:   `composer require "drupal/devel:1.x-dev"`
-   Updating an existing module: `composer update drupal/address -–with-dependencies`
-
- See the `project-base README`_ for more details.
+ - The `bin` folder contains [Drupal Console] binary.
```

Comment:
"contains the [Drupal Console] binary"
Do we also want to mention that phpunit is there?

Reply:
We should say that it contains any library binaries, such as Console, PHPUnit, Behat, etc